.github/workflows/ci.yml

```diff
@@ -3,44 +3,161 @@ name: nf-core CI
 on:
   push:
     branches:
-      - dev
+      # - dev
+      - ci_nf-test
```

Comment:
Since the PR is adding CI tests that should work on PRs made to the dev branch, this change should be reverted.

Reply:
Will do. I just removed it for this branch because I didn't want it running the ci unnecessarily until it was merged.

Reply:
done